Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use multi-stage build
+FROM golang:1.15 as builder
+
+ARG KAVANODE_VERSION
+ARG ROSETTA_KAVA_VERSION
+
+# Build kava node
+RUN git clone https://github.com/kava-labs/kava \
+    && cd kava \
+    && git checkout $KAVANODE_VERSION \
+    && make install
+
+# Build rosetta-kava service
+RUN git clone https://github.com/kava-labs/rosetta-kava \
+    && cd rosetta-kava \
+    && git fetch origin \
+    && git checkout $ROSETTA_KAVA_VERSION \
+    && make install
+
+CMD cd rosetta-kava \
```

Comment:
Updated in https://github.com/Kava-Labs/rosetta-kava/pull/10/commits/110777c79c548f618461a08d02e9243d77937d7b. Kept the `cd` though: running the script directly gives a permissions error, might be because the necessary git permissions only apply when the cmd is run from the containing directory.